Services/RequestServer/ConnectionFromClient.cpp

```diff
@@ -8,6 +8,7 @@
 
```

Comment:
commit message nit:
This seems to be _removing_ \v\f instead of allowing it, we were already "allowing" them to pass through.
Something saying "trim \v\f as well" would probably be nicer.

Reply:
Ohhh, sorry, nevermind this. we're _now_ letting \v\f to pass through.